spec/element_locator_spec.rb

```diff
@@ -459,6 +457,68 @@
 
         expect(locate_all(selector)).to eq elements.last(2)
       end
+
+      context "and xpath" do
+        before do
+          expect(Watir).to receive(:prefer_css?).and_return(false)
+        end
+
+        it "converts a leading run of regexp literals to a contains() expression" do
+          elements = [
+            element(tag_name: "div", attributes: { class: "foo" }),
+            element(tag_name: "div", attributes: { class: "foob" })
+          ]
+
+          expect_all(:xpath, "(.//div)[contains(@class, 'fo')]").and_return(elements.last(1))
```

Comment:
I feel like this is not the correct behavior. Why does it lookup `fo`? Will it iterate over these elements after?